vector/src/main/res/layout/fragment_ftue_account_created.xml

```diff
@@ -111,17 +118,46 @@
         android:textAllCaps="true"
         android:textColor="@color/element_background_light"
         android:transitionName="loginSubmitTransition"
-        app:layout_constraintBottom_toTopOf="@id/accountCreatedSpace5"
+        app:layout_constraintBottom_toTopOf="@id/ctaBottomBarrier"
         app:layout_constraintEnd_toEndOf="@id/ftueAuthGutterEnd"
         app:layout_constraintStart_toStartOf="@id/ftueAuthGutterStart"
         app:layout_constraintTop_toBottomOf="@id/accountCreatedPersonalize" />
 
+    <androidx.constraintlayout.widget.Group
+        android:id="@+id/takeMeHomeButtonGroup"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:constraint_referenced_ids="accountCreatedTakeMeHomeCta" />
+
+    <Button
+        android:id="@+id/accountCreatedTakeMeHomeCta"
+        style="@style/Widget.Vector.Button.Login"
```

Comment:
We have a handy `Widget.Vector.Button.CallToAction` but it's used much (it's quite new). I am wondering if we should at some point use this style a bit more.

Reply:
Sounds like a plan! (that was my original intention to make you think that 👿 )

Reply:
https://github.com/vector-im/element-android/issues/5515